service/history/workflowTaskHandlerCallbacks.go

```diff
@@ -468,8 +476,6 @@ func (handler *workflowTaskHandlerCallbacksImpl) handleWorkflowTaskCompleted(
 		activityNotStartedCancelled bool
 		newMutableState             workflow.MutableState
 	)
-	// hasPendingUpdates indicates if there are more pending updates (excluding those which are accepted/rejected by this workflow task).
-	hasPendingUpdates := weContext.UpdateRegistry().HasPending(request.GetMessages())
```

Comment:
I am super happy see this getting removed.

Reply:
Ha. If only it didn't break :facepalm: